iron-multi-selectable.html

```diff
@@ -89,6 +87,19 @@
       }
     },
 
+    _observeItems: function(node) {
+      var observer = new MutationObserver(function() {
+        if ((this.multi && this.selectedValues && this.selectedValues.length) || this.selected != null)  {
```

Comment:
Instead of completely overriding the implementation of this method _just_ to change the boolean logic here, we should probably factor out the boolean to an accessor. This way, overriding implementation can replace the accessor without having to re-implement the `MutationObserver` part of this method. Such an approach would have the bonus effect of making this conditional much easier to read.